ddtrace/appsec/utils.py

```diff
@@ -13,13 +22,80 @@ def _appsec_rc_features_is_enabled():
 
 def _appsec_rc_capabilities():
     # type: () -> str
-    """return the bit of the composed capabilities in base64
+    r"""return the bit representation of the composed capabilities in base64
```

Comment:
Is this change specific to IP blocking? Can we introduce it separately?

Reply:
Is needed for the feature registration, could be introduced separately however.